source/application/CLS/CLSAppController.h

```diff
@@ -12,8 +12,8 @@ class CLSAppController : public AMAppController
 public:
 	enum CLSBeamlineID
 	{
-		OtherFacility = 1,    //reserved for AMAppControlelr
-		PresevedBeamline = 2, //reserved for AMAppControlelr
+		BlankFacility = 1,            //reserved for AMAppControlelr
+		PreserverdALS801Beamline = 2, //reserved for AMAppControlelr
```

Comment:
Nope.  At the very beginning the data we imported (using the very deprecated AMImporter) was from the ALS.